### PR TITLE
Display a warning dialog when the FirebaseDB component is used.

### PR DESCRIPTION
**Pull request by jisqyv**
Change-Id: I83175a2a311a3ca874e8d02d060ceecc4316c5c8

### CODE REVIEW

**Comment by jisqyv** [comment]
@halatmit @afmckinney Please Review

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockFirebaseDB.java [hunk 1]
```diff
@@ -126,22 +126,20 @@ public void onFailure(Throwable caught) {
...
+  /**
+   * Called when the component is dropped in the Designer window
+   * we give a warning that firebase is still experimental.
+   */
+
+  @Override
+  public void onCreateFromPalette() {
+    if (!warningGiven) {
+      warningGiven = true;
+      Ode.getInstance().warningDialog("Warning!", "FirebaseDB is an experimental feature " +
```
Comment (josmas) [inline appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockFirebaseDB.java]: depending on how long this is going to stay in the codebase, should it be internationalised?

**Comment by jisqyv** [comment]
@josmas I should internationalize it. I'll take care of it.

**Comment by jisqyv** [comment]
@josmas I've added the Internationalization

**Comment by josmas** [comment]
LGTM

**Comment by jisqyv** [comment]
Moving to Gerrit

**Comment by jisqyv** [comment]
Squashed

**Pull request merged**